docs/tool/manifest.js

```diff
@@ -72,15 +72,15 @@ function getDataManifest( parsedNamespaces ) {
 	return [ {
 		title: 'Data Package Reference',
 		slug: 'data',
-		markdown_source: `${ baseRepoUrl }/docs/data/README.md`,
+		markdown_source: `${ baseRepoUrl }/docs/designers-developers/developers/data/README.md`,
 		parent: null,
 	} ].concat(
 		Object.values( parsedNamespaces ).map( ( parsedNamespace ) => {
 			const slug = `data-${ kebabCase( parsedNamespace.name ) }`;
 			return {
 				title: parsedNamespace.title,
 				slug,
-				markdown_source: `${ baseRepoUrl }/docs/data/${ slug }.md`,
+				markdown_source: `${ baseRepoUrl }/docs/designers-developers/data/${ slug }.md`,
```

Comment:
Looks like this should be:

```suggestion
				markdown_source: `${ baseRepoUrl }/docs/designers-developers/developers/data/${ slug }.md`,
```